packages/x-date-pickers/package.json

```diff
@@ -55,6 +55,9 @@
     "rifm": "^0.12.1"
   },
   "peerDependencies": {
+    "@babel/runtime": "^7.16.3",
```

Comment:
On the core, `@babel/runtime` is a dep, not a peer dep

I wonder why we never needed it on the data grid packages...
@mnajdova do you know what it is being used for on the core ?

Reply:
Would work for me too to put `@babel/runtime` into dependencies. But it needs to be defined somewhere.

Reply:
Looks like most libraries would put that into the dependencies section. Went ahead and changed it in the PR. Also I made the emotion imports optional, as mui can also be used with styled-components.